src/main/resources/spring/camel-context.xml

```diff
@@ -29,11 +29,25 @@
         </property>
     </bean>
 
+    <bean id="zipAggregationStrategy" class="org.apache.camel.processor.aggregate.zipfile.ZipAggregationStrategy">
+        <constructor-arg index="0" value="false"/>
+        <constructor-arg index="1" value="true"/>
+    </bean>
+
+    <bean id="dateUtils" class="org.jboss.xavier.utils.DateUtils"/>
+    <bean id="utils" class="org.jboss.xavier.utils.Utils"/>
+
     <camelContext id="camel" xmlns="http://camel.apache.org/schema/spring">
 
         <interceptFrom id="interceptor-username" uri="rest:*">
             <to uri="direct:check-authenticated-request" />
         </interceptFrom>
+        <interceptFrom id="interceptor-authorized-request" uri="rest:*">
```

Comment:
Could this interceptor be defined only for `uri="rest:/administration/*"`  in order to remove the `when` condition?
If not, i see no reason to have this filter and the `when` block could be moved into the previous `interceptor-username` interceptor safely, right?

Reply:
`uri="rest:/administration/*"` does not work so I decided to move the code to "interceptor-username"